src/lib/components/bottom-nav-links.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+  import NavigationItem from '$lib/holocene/navigation/navigation-item.svelte';
+  import type { NavLinkListItem } from '$lib/types/global';
+
+  export let open = false;
+  export let linkList: NavLinkListItem[];
+</script>
+
+{#if open}
+  <div class="flex h-full flex-col justify-start gap-4">
+    {#each [...linkList].reverse() as item}
```

Comment:
I think we could just do this with css with `flex-col-reverse`

Reply:
Good call.